### PR TITLE
Add more hints about template editing

### PR DESCRIPTION
**Pull request by greg0ire**
There are way too many people that don't seem to understand that this
template is supposed to be changed. Add more hints for them.

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
LGTM for me.

cc @sonata-project/contributors for other reviews.

**Comment by OskarStark** [comment]
LGTM 👍

**Comment by core23** [comment]
Think we can merge this @Soullivaneuh

**Comment by OskarStark** [comment]
ping @Soullivaneuh

File: project/.github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -7,15 +7,15 @@
...
 
 <!--
     Specify which issues will be fixed/closed.
     Remove it if this is not related.
 -->
 
-Closes #
-Fixes #
+Closes #{put_issue_number_here}
+Fixes #{put_issue_number_here}
```
Comment (core23) [inline project/.github/PULL_REQUEST_TEMPLATE.md]: Do we need both? `Closes` should be enough.

Comment (greg0ire) [inline project/.github/PULL_REQUEST_TEMPLATE.md]: I think there is indeed no difference, I'll fix it

**Comment by core23** [comment]
Can we merge this @Soullivaneuh ?

**Pull request merged**